packages/sdk/src/across/constants.ts

```diff
@@ -8,6 +8,10 @@ export const FAST_ETH_GAS = 273519;
 export const FAST_ERC_GAS = 281242;
 export const FAST_UMA_GAS = 305572;
 
+export const SPEED_UP_ETH_GAS = 195288;
```

Comment:
I really hate hardcoding these, especially when its so easy to estimate gas via contract.methods.estimateGas

Reply:
ye, it's not ideal. the main argument for doing it like this is to make the bots compute gas the same way the front end does.

Reply:
Yeah agreed lets leave for now